meta-xt-control-domain/recipes-guest/domz/files/domz-generic.cfg

```diff
@@ -0,0 +1,21 @@
+# =============================================== #
+# Configuration of domain with Zephyr OS as guest #
+# =============================================== #
+
+seclabel='system_u:system_r:domz_t'
+
+# Guest name
+name="DomZ"
+
+# Kernel image to boot
+kernel="/usr/lib/xen/boot/zephyr-domz"
+
+# Number of VCPUS
+vcpus=1
+
+# Initial memory allocation (MB)
+memory=16
+
+gic_version="v2"
```

Comment:
Are you sure that Zephyr does not support GICv3? Please mind that this is `meta-xt-common` layer, which is used by all platforms.

Reply:
I just used the first mentioned config from https://docs.zephyrproject.org/latest/boards/arm64/xenvm/doc/index.html.

As I understand GIC version should be specified in this config line and in board `west build -b xenvm_gicv3 ...`.
Seem that I need to pass `board` variable from yaml down to the recipe to select the proper config.

Reply:
`gic_version` is set to `default` and successfully tested on HW with GIC v2.